spec/features/task_managements_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.feature "TaskLogs", type: :feature do
+  before do
+    Capybara.default_driver = :selenium
+    @task1 = create(:task_management, task_desc: "Iron these clothes for me")
```

Comment:
Since you can use any string for the task description, please use https://github.com/stympy/faker#fakerlorem. And make it a variable so you can declare once and reuse. Something like, task1_desc = Faker::Lorem.sentence. Then call it everywhere you need it.